pusher/src/Controller/IoSocketController.ts

```diff
@@ -610,6 +611,11 @@ export class IoSocketController {
                         client,
                         message.getLockgrouppromptmessage() as LockGroupPromptMessage
                     );
+                } else if (message.hasMapeditormodifyareamessage()) {
+                    socketManager.handleMapEditorModifyAreaMessage(
+                        client,
+                        message.getMapeditormodifyareamessage() as MapEditorModifyAreaMessage
+                    );
```

Comment:
Each time we add a new edition feature, we will need to add the message in the pusher here (and create a new method in the SocketManager), then add a method in the back to forward this to the map-storage.

This is going to be a giant pain in the a@#.

We can probably do simpler.

We could wrap the MapEditorModifyAreaMessage in `messages.proto` into another type: `EditMapMessage`.

This way, the pusher and back could simply forward the `EditMapMessage` to the map-storage.
We would have to do this only once (and not once per edition message!)